### PR TITLE
enhance people view

### PR DESCRIPTION
**Pull request by mbwatson**
this PR aims to list people on the the `/people` view alphabetically. a couple related efforts are folded in here:

- sort people alphabetically at fetch-time
- fix ood grouping/filtering
- conditionally pass and render generic avatar

### CODE REVIEW

**Review by Woozl — APPROVED** [review]
:+1:

File: pages/people/index.js [hunk 1]
```diff
@@ -107,8 +107,7 @@ const HorizontalLettersMenu = ({ linkedLetters }) => {
 }
 
 /*
- * people are coming into this component from
- * fetchStrapiPeople with this shape:
+ * people are coming into this component with this shape:
```
Comment (suejinkim20) [inline pages/people/index.js:101]: should we also filter the people array by the active field? currently it's pulling in both active and inactive staff members for the staff page

Comment (mbwatson) [reply pages/people/index.js:101]: ooh yes. thanks for catching that!

File: lib/dashboard/people.js [hunk 2]
```diff
@@ -6,20 +6,21 @@ import {
...
+    .map(person => ({
+      ...person,
+      // give person our generic avatar if no photo exists
+      photo: person.photo || genericAvatar.src,
+    }))
+    .sort((p, q) => {
+      return p.firstName.localeCompare(q.firstName) ? p.lastName.localeCompare(q.lastName) : -1
+    });
+
+  const ood = people.filter(person => person.teams.some(t => t.name === 'Office of the Director'));
```
Comment (mbwatson) [inline lib/dashboard/people.js]: use dashboard uid or slug

**Comment by mbwatson** [comment]
rebased and addressed conflicts

File: lib/dashboard/people.js [hunk 3]
```diff
@@ -3,20 +3,26 @@ import { getProjectRelations, getDivisionRelations } from "./utils";
...
-    if (person.teams.length > 0) {
-      person.teams.forEach((team) => {
-        if (team.data === "ood") {
-          oodList.push(person);
-        }
-      });
-    }
-  });
+  people = people
+    .filter((person) => person.isActive)
```
Comment (mbwatson) [inline lib/dashboard/people.js:9]: filtering out inactive folks here

**Pull request merged**